galite-core/src/main/kotlin/org/kopi/galite/form/VFixnumField.kt

```diff
@@ -282,10 +284,10 @@ class VFixnumField(private val digits: Int,
    */
   fun setMaxScale(scale: Int) {
     // dynamic maxScale mustn't exceed the maxScale defined in the field declaration (fieldMaxScale).
-    if (scale > fieldMaxScale) {
+    if (scale > maxScale) {
```

Comment:
maxScale --> fieldMaxScale 

then in  fun setScale(record: Int, scale: Int) { --> change fieldMaxScale to maxScale